### PR TITLE
doc / update installation gateway from source

### PR DESCRIPTION
**Pull request by glenntomas**
**Before submitting this PR, please make sure**:

- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**A description of the changes proposed in the pull request**:

- Updates and changes comes from QA @RC-13 and @PtrckM 

- Update `gateway.env` to `global_conf.yml`

image

- Update node version and gif
- Add nvm link in tip callout

image

- Update yarn installation and gif
- Update sample environment and gif

image

- Remove yarn run dev
- Update yarn debug
- Update yarn start and gif
- Update `.env` to `global_conf.yml`

image

- Updated the list of `ecr20_tokens_kovan.json` and the path of the file

image

**Tests performed by the developer**:

**Tips for QA testing**:

### CODE REVIEW

**Review by mifeng — APPROVED** [review]

**Comment by mifeng** [comment]
Since this is a correction to the published documentation only, we can merge this PR to `master` after 1 more approval. @dennisocana we should cherry-pick this commit onto `development`

**Comment by dennisocana** [comment]
@mifeng will do. Just an FYI similar to our hotfix process, we will revert this change from the `master` branch on the next release to avoid merge conflicts.

File: documentation/docs/installation/gateway.md
Comment (dennisocana) [inline documentation/docs/installation/gateway.md:52]: Incorrect file name.

File: documentation/docs/installation/gateway.md
Comment (dennisocana) [inline documentation/docs/installation/gateway.md:147]: Incorrect file name.

File: documentation/docs/installation/gateway.md
Comment (dennisocana) [inline documentation/docs/installation/gateway.md:152]: Incorrect file name.

**Review by dennisocana — CHANGES_REQUESTED** [review]

**Review by dennisocana — APPROVED** [review]

**Pull request merged**
